app/src/androidTest/java/com/github/orkest/ViewModel/feed/PostViewModelTest.kt

```diff
@@ -70,6 +70,35 @@ class PostViewModelTest {
         assert(posts[0].postDescription == "Recent1")
     }
 
+    @Test
+    fun getRecentPostsReturnsPostsInDescendingOrderOfDate(){
+        Constants.CURRENT_LOGGED_USER = "DummyUser"
+        postViewModel.updateSong(Constants.DUMMY_RUDE_BOY_SONG)
+        postViewModel.updatePostDescription(TextFieldValue("Recent1"))
```

Comment:
Very good test. You could maybe add tests about edge cases like when there are no posts to fetch or maxNbDays parameter is higher than the constant MAX_NB_DAYS